devTools/Dockerfile-php71-xdebug

```diff
@@ -1,6 +1,7 @@
 FROM php:7.1-cli
 
-RUN pecl install xdebug && docker-php-ext-enable xdebug
+RUN pecl install xdebug && docker-php-ext-enable xdebug && \
+    docker-php-ext-install -j$(nproc) bcmath
```

Comment:
As in the previous comment, from my point of view - Infection does not depend on `bcmath`, thus it's not needed.

`\function_exists($functionName);` in this condition already covers all we need:

```php
public function mutatesNode(...)
{
    ...
    return isset($this->converters[$functionName]) && \function_exists($functionName);
}
```

Reply:
change reverted